crates/interpreter/src/instructions/opcode.rs

```diff
@@ -829,6 +847,16 @@ const fn make_gas_table(spec: SpecId) -> [OpInfo; 256] {
     table
 }
 
+pub fn make_instruction_table<H: Host, SPEC: Spec>() -> [Instruction<H>; 256] {
```

Comment:
I think this and `instruction` can be made `const`

Reply:
Tried it and the problem with `Host` error is something like "cant use parent generic" or something in that tone.